canvas_modules/common-canvas/src/common-canvas/keyboard-utils.js

```diff
@@ -166,6 +167,26 @@ export default class KeyboardUtils {
 		return d3Event.key === UP_ARROW_KEY;
 	}
 
+	static focusSubObject(d3Event) {
+		return this.isMetaKey(d3Event) && d3Event.key === ONE_KEY;
```

Comment:
There seems to be a conflict with this key combo in Mac. `cmd + 1`, if you have multiple tabs in the browser, will take you to the first tab in the browser. Subsequently `cmd + 2` will take you to the second opened tab

Reply:
I changed it to `Alt (Option on the Mac) + Tab` to initially move focus to the sub-objects in a node.